spec/install/deploy_spec.rb

```diff
@@ -263,4 +263,28 @@
       should_be_installed "rack 1.0.0"
     end
   end
+
+  context "with path in Gemfile and packed" do
+    it "works fine after bundle package and bundle install --local" do
+      build_lib "foo", :path => lib_path("foo")
+      install_gemfile <<-G
+      gem "foo", :path => "#{lib_path("foo")}"
+      G
+
+      bundle :install
+      should_be_installed "foo 1.0"
+      bundle "package --all"
+      expect(bundled_app("vendor/cache/foo")).to be_directory
+
+      bundle "install --local"
```

Comment:
It'd be great to get some further assertions here (maybe on output?) rather than just exitstatus.